images/permissions.md

```diff
@@ -1,10 +1,10 @@
 # Details for attribution and permission for each of the images
 
-- `100-0-calibration.png` - Patricia Lopez-Garcia
+- `100-0-calibration.png` - Patricia Lopez-Garcia/Tania Morales
 - `AanderaSensors.png` - XXX, Aanderaa data instruments
```

Comment:
Hi @patricialg thanks for updating the figures. Please always decide on one name for the image credits.